src/main/java/com/notably/logic/suggestion/SuggestionEngineImpl.java

```diff
@@ -29,7 +30,7 @@
 public class SuggestionEngineImpl implements SuggestionEngine {
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
 
-    private static final List<String> COMMAND_LIST = List.of("new", "edit", "delete", "open", "help", "exit");
+    private static final List<String> COMMAND_LIST = List.of("new", "edit", "delete", "open", "help", "exit", "search");
```

Comment:
Actually, this `COMMAND_LIST` should use the constants `DeleteSuggestionCommandParser.COMMAND_WORD`, `SearchSuggestionCommandParser.COMMAND_WORD`, etc.